bad_code.java

```diff
@@ -45,9 +46,9 @@ public void doSomethingElse() {
         for (int i = 0; i < someNumber; i++) {
```

Comment:
Replace the reference to the old variable name `someNumber` with the new variable name `number` to avoid compilation errors. [important]